drivers/wifi/infineon/airoc_wifi.h

```diff
@@ -6,16 +6,44 @@
  */
 
 #include <whd_buffer_api.h>
-#include <zephyr/sd/sd.h>
-#include <zephyr/sd/sdio.h>
 #include <zephyr/drivers/gpio.h>
 #include <zephyr/net/wifi_mgmt.h>
+
+#ifdef CONFIG_AIROC_WIFI_BUS_SDIO
+#include <zephyr/sd/sd.h>
+#include <zephyr/sd/sdio.h>
+#endif
+#ifdef CONFIG_AIROC_WIFI_BUS_SPI
```

Comment:
```suggestion
#elif defined(CONFIG_AIROC_WIFI_BUS_SPI)
```
nit/suggestion: Would having the above suggestion making it clearer that the driver supports either SDIO, or SPI, but not both? 

This would match the implementation in drivers/wifi/infineon/airoc_wifi.c .

Reply:
If multiple instances cannot coexist due to essential(hardware) constraints, it is better, but because it's not, in this case, it has no benefit other than making it easier to understand, so I think it's fine to leave it as is.
If you want to fix it, it would be better to add support for multiple instances, but that doesn't need to be included in the scope of this PR.